bindata/network/kuryr/003-config.yaml

```diff
@@ -70,7 +70,7 @@ data:
     auth_url = {{ $AuthInfo.AuthURL }}
     insecure = {{ .OpenStackInsecureAPI }}
     token = {{ default "\"\"" $AuthInfo.Token }}
-    password = {{ default "\"\"" $AuthInfo.Password }}
+    password = {{ default "\"\"" (escapeCharacters $AuthInfo.Password) }}
```

Comment:
I believe we should escape other vars too, in particular: `token`, `username`, `project_domain_name`, `project_name`, `user_domain_name`, `region_name`.

Reply:
Indeed. When posting directly to the API it's possible to set those values with '$'